src/helperFunctions/topicFormat.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
This file will be a temporary fix until we fix the back end  - will delete in the future